connectivity/FEATURE_BLE/source/common/ble_trace_helpers.h

```diff
@@ -54,11 +54,7 @@ static inline char* to_string(const UUID& uuid)
     static char strbuf[max_num_row][UUID::LENGTH_OF_LONG_UUID + /* Number of hyphen delimiters =*/ 4];
     static uint8_t idx = 0;
 
-    if (idx == max_num_row) {
-        idx = 0;
-    } else {
-        idx++;
-    }
+    idx == max_num_row ? idx = 0 : idx++;
```

Comment:
the compiler will produce the same code, another matter is whether using ternary operator for side effects instead of value is good practice - I leave the field to our architects